src/style.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  list-style: none;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

can be simplify, check in other places also
```suggestion
  background: url(./images/star.svg) no-repeat center;
```